setup.cfg

```diff
@@ -44,8 +44,14 @@ exclude = venv/*,var/*
 
 # Ignore on a per file basis:
 # F401 - module imported but not used
+# E501 - line too long, exceeds max-line-length
+# F821 - undefined name
 
 ignore = E203,E731,W503,W605
 per-file-ignores =
   #  Keep the unused imports as they are used to route the rest service
   codalab/server/rest_server.py: F401
+  #  Keep the previous format of documentation
```

Comment:
This file isn't documentation. It's a script.

Reply:
I know, but I think the text here will be written to the markdown. That's why I worded like this